src/ws-dropdown/dropdown-menu-item.js

```diff
@@ -113,16 +113,20 @@ export class DropdownMenuItem extends Component {
         if (this.state.selected) {
           this.props.handle('change', null);
         } else {
-          this.state.selected = true;
-          this.state.stored = true;
+          this.setState({
+            ...this.state,
```

Comment:
I thought setState is merging the state anyways so this line would be obsolete or am I wrong?

Reply:
yes.  i initially did it just for compat vs. the original, but i think you're right.  i'll do a quick check then update